src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -3,29 +3,29 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import unittest
 import os
-import inspect
 import subprocess
 import json
 import shlex
-import tempfile
-import shutil
 import logging
-import six
-import vcr
+import inspect
+import sys
+
+from azure_devtools.scenario_tests.base import LiveTest
+from azure_devtools.scenario_tests.base import ScenarioTest as DevToolsScenarioTest
```

Comment:
Is `ScenarioTestBase` OK? I feel like I've seen that naming convention more often.